src/Factories/FunFunFactory.php

```diff
@@ -862,7 +710,16 @@ private function newHonorificValidator(): AllowedValuesValidator {
 	}
 
 	private function getHonorifics(): Honorifics {
-		return $this->pimple['honorifics'];
+		return $this->createSharedObject( Honorifics::class,
```

Comment:
Should we move this into the content repo?

Reply:
Yep, that would improve things. Maybe not in this PR, but I'll work on this.